Atlanta.md

```diff
@@ -0,0 +1,3 @@
+# Atlanta, GA
+## Where to eat 
```

Comment:
@EprisBlank What's your favorite place to eat ? I'd like to see a hometown restaurant not really a franchise.